FreeRTOS_IP.c

```diff
@@ -1212,14 +1202,13 @@ BaseType_t FreeRTOS_IPInit( const uint8_t ucIPAddress[ ipIP_ADDRESS_LENGTH_BYTES
     configASSERT( xNetworkEventQueue == NULL );
     configASSERT( xIPTaskHandle == NULL );
 
-    if( sizeof( uintptr_t ) == 8 )
     {
```

Comment:
Why not get rid of these curly braces?

Reply:
Indeed, they don't serve a purpose, I removed them in #357.